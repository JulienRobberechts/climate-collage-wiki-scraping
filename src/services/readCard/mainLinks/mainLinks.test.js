/**
 * @jest-environment node
 */
const { getCardRelations } = require('./mainLinks');
const { getCardsFrReferenceByCardNum } = require('../../data-access/cardsRepo')

describe('getCardRelations', () => {
  it('card 1', async () => {
    const cardNum = 1;
    const card = await getCardsFrReferenceByCardNum(cardNum);
    const actualLinks = await getCardRelations(card.wikiId, `getCardRelations test`);
    expect(actualLinks.causes).toMatchSnapshot();
    expect(actualLinks.effects).toMatchSnapshot();

  });
});
