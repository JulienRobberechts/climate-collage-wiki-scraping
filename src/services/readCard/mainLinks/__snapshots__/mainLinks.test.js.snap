// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCardRelations card 1 1`] = `Array []`;

exports[`getCardRelations card 1 2`] = `
Array [
  2,
  3,
  4,
  8,
]
`;
